### PR TITLE
Hide Tomcat version on error pages

### PR DESCRIPTION
**Pull request by aren-pulid0**

### CODE REVIEW

**Review by frantsao — APPROVED** [review]

**Review by blalop — APPROVED** [review]

**Comment by aren-pulid0** [comment]
Added a flag to activate/deactivate this feature :)

File: templates/server.xml.j2 [hunk 1]
```diff
@@ -136,7 +136,11 @@
         <Valve className="org.apache.catalina.valves.AccessLogValve" directory="{{ path }}"
                prefix="localhost_access_log" suffix=".txt"
                pattern="%h %l %u %t &quot;%r&quot; %s %b" />
-
+        
+        {% if tomcat_show_server_info_on_error_page == false %}
```
Comment (vicsufer) [inline templates/server.xml.j2:140]: A better condition would be`if not tomcat_show_server_info_on_error_page`

Comment (aren-pulid0) [reply templates/server.xml.j2:140]: hmm i like it to be explicit, that would make any value on `tomcat_show_server_info_on_error_page` as valid

Comment (vicsufer) [reply templates/server.xml.j2:140]: if wont, that would be `tomcat_show_server_info_on_error_page is defined`

**Pull request merged**